### DISCUSSION TITLE
Blue Scsi won't work unless another SCSI Device is connected

### DISCUSSION THREAD

**Discussion by fragmaxsr** [erichelgeson/BlueSCSI-v1#74] [Q&A]
I have a SCSI2SD connected with no SD card and the Blue SCSI with a micro sd card. My mac classic II will boot. But if I remove the SCSI2SD card and just use the BLUESCSI alone it won't boot.  The red light is on.  I have tried the SCSI2SD external and The BLUE  SCSI internal and I have swapped them around and BLUESCSI external and SCSI2SD internal and both way work if they are both connected. They do not need additional power.   I do have it terminated with both Jumpers.  I have rechecked all my soldering.  I have a second Mac Classic II and the same thing happens.  I also have a Mac SE, again same thing.  As long as the SCSI2SD is connected it will work.  I have tried renaming the file HD00_512.hda HD10_512.hda HD30_512,hda. Still nada.   Any help is appreciated.

**Comment by erichelgeson** [comment]
Are you using a known good/working image or blank from the Usage page of the wiki? Also test the voltages to ensure they are near 5v. If they are too low proper termination on the BlueSCSI will not work. Last take a look at the Troubleshooting wiki page and ensure you don't skip any troubleshooting steps.

**Reply by fragmaxsr** [reply]
Tried everything.  I have another kit, I am going to try that one and see if its me or this kit. Thanks for help.
